en/Advanced topics/Drag and Drop.md

```diff
@@ -14,16 +14,16 @@ See [[Working with multiple notes#5 Panes can be rearranged by dragging|dragging
 
 ### Drop destinations
 
-- You can drop a file on a pane header to open the file there. Hold `Alt` (or `Shift` on MacOS) to drop anywhere on the pane.
+- You can drop a file on a pane header to open the file there. Hold `Alt` (or `Shift` on macOS) to drop anywhere on the pane.
 - You can drop a file on a folder in the [[file explorer]] to move the file there.
-- You can drop a file into an editor to insert it as a link. The generated link also follows your preferences, such as relative paths, or using markdown links.
+- You can drop a file into an editor to insert it as a link. The generated link also follows your preferences, such as relative paths, or using Markdown links.
 - You can drop a file on the [[Starred]] to star it.
 
 ### Dragging from outside Obsidian
 
 - You can drag any HTML content from the browser into Obsidian to automatically convert them into Markdown
-- You can drag and drop any files from your native file explorer to have Obsidian import (make a copy of) those files in your vault's attachment folder and insert them as internal links. Hold `Ctrl` on Windows/Linux or `Alt` on Mac to create `file:///` absolute links to those files instead of importing a copy.
+- You can drag and drop any files from your native file explorer to have Obsidian import (make a copy of) those files in your vault's attachment folder and insert them as internal links. Hold `Ctrl` on Windows/Linux or `Alt` on macOS to create `file:///` absolute links to those files instead of importing a copy.
```

Comment:
```suggestion
- You can drag and drop any files from your native file explorer to have Obsidian import (make a copy of) those files in your vault's attachment folder and insert them as internal links. Hold `Ctrl` on Windows/Linux or `Option` on macOS to create `file:///` absolute links to those files instead of importing a copy.
```